rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -176,10 +165,37 @@ void getActiveFederatorBtcPublicKey_withNegativeIndex_throwsIndexOutOfBoundsExce
         void getActiveFederatorBtcPublicKey_withIndexGreaterThanGenesisFederationSize_throwsIndexOutOfBoundsException() {
             int genesisFederationSize = genesisFederation.getSize();
             assertThrows(
-                IndexOutOfBoundsException.class, () -> 
+                IndexOutOfBoundsException.class, () ->
                 federationSupport.getActiveFederatorBtcPublicKey(genesisFederationSize)
             );
         }
+
+        @Test
+        @Tag("getActiveFederatorPublicKeyOfType")
+        void getActiveFederatorPublicKeyOfType_returnsFederatorPublicKeysFromGenesisFederation() {
+            BtcECKey federatorFromGenesisFederationBtcPublicKey = genesisFederation.getBtcPublicKeys().get(0);
+            ECKey federatorFromGenesisFederationRskPublicKey = RskTestUtils.getRskPublicKeysFromFederation(genesisFederation).get(0);
+            ECKey federatorFromGenesisFederationMstPublicKey = getMstPublicKeysFromFederation(genesisFederation).get(0);
```

Comment:
One in RskUtils and the other in the class?

Reply:
i thought that the mst method will only be used here (for now and for a long time probably, since we don't have any mst-related things in the project).
On the other side, the rsk one could be useful in other places, and since i had already created the RskTestUtils class i thought it was a good place to have it.

Does it make sense?

Reply:
solved